src/bodywork/k8s/auth.py

```diff
@@ -209,8 +210,8 @@ def setup_workflow_service_account(namespace: str) -> None:
         )
 
 
-def setup_job_and_deployment_service_accounts(namespace: str) -> None:
-    """Setup a jobs-and-deployments service-account with required roles.
+def setup_job_and_deployment_service_account(namespace: str) -> None:
```

Comment:
Maybe we should refactor this to `setup_stages_service_account`. My own naming choices are confusing me!

Reply:
Agreed!